plugin/src/main/java/org/opensearch/ml/action/undeploy/TransportUndeployModelAction.java

```diff
@@ -104,115 +85,9 @@ protected MLUndeployModelNodesResponse newResponse(
         List<MLUndeployModelNodeResponse> responses,
         List<FailedNodeException> failures
     ) {
-        if (responses != null) {
```

Comment:
A simpler way to fix https://github.com/ylwu-amzn/ml-commons/commit/bdca03030b2694fb804fa670a823113070dfbdfd